Assets/Scenes/Navigation.prefab

```diff
@@ -0,0 +1,320 @@
+%YAML 1.1
```

Comment:
Can you explain what this prefab is for? And see [this discussion](https://cvmlp.slack.com/archives/C05HWV5UZ1A/p1707154210737549?thread_ts=1707148506.826349&cid=C05HWV5UZ1A). I wonder if we can convert this to a piece of readable code.

Reply:
This shares the navmesh and floor collision objects between the two scenes.

This could be created dynamically via script.

Reply:
I'd strongly prefer that we create dynamically via script. We're setting a bad example for how to do reusability/readability if we do it this way.

Reply:
This was changed to separate `CollisionFloor` and `NavmeshHelper` scripts that are dynamically created.